docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.9" # optional since v1.27.0
```

Comment:
Let me know if you'd rather not have docker-compose.yml in the project

Reply:
That's fine, we need to support the 3 official adapters and having a docker-compose helps.